frontend/assets/stylesheets/components/_bundle-offerings-a.scss

```diff
@@ -0,0 +1,79 @@
+.offering-column,
+.offering-column--left-column{
+    width: 300px;
+    display: inline-block;
+    vertical-align: top;
+    background-color: #d4ebf8;
+    margin-top: 24px;
+}
+
+.offering-column--left-column {
```

Comment:
I think it would be better to ditch the `offering-column--left-column` class and use [`.offering-column:first-child`](https://developer.mozilla.org/en-US/docs/Web/CSS/:first-child)